docs/launchpad/getting-started/permissions.md

```diff
@@ -6,30 +6,27 @@ sidebar_position: 3
 
 ## Location Permission
 
-![Location Permission 1](img/permissions-location-1.jpg)
-
-![Location Permission 2](img/permissions-location-2.jpg)
-
-
-The moment you login to the application the first permission that is required is for location services. This permission
-is required by the app to show the **user** location and **not the drone**. As you can see the app presents you with two
+After logging into to the application the first permission that is required is for location services. This permission is
+required by the app to show the **user** location and **not the drone**. As you can see the app presents you with two
 options.
 
-1. `Precise` - Ideal conditions can pinpoint you down to a meter.
-2. `Approximate` - Your location can be within 100 meters.
+![Location Permission](img/permissions-location.jpg)
+
+- `Precise` - Ideal conditions can pinpoint you down to a meter.
+- `Approximate` - Your location can be within 100 meters.
 
 The user has three options to select from:
```

Comment:
This line can be modified to `Additionally user has to select location usage options`